### PR TITLE
Add support for subSeconds alarms

### PR DESCRIPTION
**Pull request by angelnu**
I need to use this to add support to ms sleeps in the STM32LowPower library. This addresses #25

### CODE REVIEW

**Comment by fpistm** [comment]
Hi @angelnu 
I guess you would mean it addresses #17 ?

**Comment by angelnu** [comment]
actually both...

**Comment by fpistm** [comment]
Ok you would say #23 from STM32LowPower....
You wrote #25 in your first post...

File: library.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name=STM32duino RTC
-version=1.0.3
+version=1.0.4
```
Comment (fpistm) [inline library.properties:2]: Please remove this change. This is up to the maintainers to define the next version.

**Comment by angelnu** [comment]
I will remove it - please notice that the low-power package would depend on this PR.

**Comment by fpistm** [comment]
> I will remove it - please notice that the low-power package would depend on this PR.

I saw anyway it is the same for STM32 Low Power and I will think about removing dependency....

**Comment by angelnu** [comment]
version change removed

**Pull request merged**